src/style.css

```diff
@@ -1,3 +1,80 @@
+:root {
+  --blue-color: #00acdc;
+  --black-color: #000;
+  --padding-size: 50px;
+}
+
+* {
+  margin: 0;
+  padding-inline-start: 0;
+}
+
 body {
   margin: 0;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+}
+
+.container {
+  display: flex;
+  align-items: center;
+}
+
+.spacer {
+  flex-grow: 1;
+}
+
+header {
+  padding-left: var(--padding-size);
+  padding-right: var(--padding-size);
+}
+
+ul {
+  list-style-type: none;
+  display: flex;
+  align-items: center;
+  position: relative;
+  margin: 0;
+}
+
+.is-active span {
+  position: relative;
+  display: inline-block;
+  color: var(--blue-color);
+}
+
+.is-active span::after {
+  content: "";
+  position: absolute;
+  left: 0;
+  bottom: -23px;
+  width: 100%;
+  height: 4px;
+  border-radius: 2px;
+  background-color: var(--blue-color);
+}
+
+li {
+  padding-right: 20px;
+}
+
+li:last-child {
+  padding-right: 0;
+}
+
+li a {
```

Comment:
I think you should consider using class selectors. Right now you are applying those styles for all `a` tags on your website